she_codes_news/news/templates/news/index.html

```diff
@@ -1,21 +1,60 @@
-    <h2>Latest News</h2>
+{% extends "base.html" %}
+    {% load static %}
+
+    {% block stylesheets %}
+    <link rel="stylesheet" type="text/css" href="{% static 'news/styles.css' %}">
+    {% endblock %}
+
+    {% block content %}
+
+    <div id="news-text">
+
+    <form method="GET" style="display: flex; flex-direction: column" >
+        <div class="search-inputs">
+        <label class="search-label">
+            {{form.search.label}}   
+        </label>
+        {{form.search}} 
+        <label class="search-label">
+            {{form.author.label}}   
+        </label>
+
+        {{form.author}} 
+        <label class="search-label">
+            {{form.categories.label}}   
+        </label>
+        {{form.categories}} 
+        </div>
```

Comment:
you should have been able to do this in a slightly better way, if you're interested we can work through this at some point later.